packages/db/src/schema.ts

```diff
@@ -743,6 +743,32 @@ export const apExamReward = pgTable("ap_exam_reward", {
   coursesGranted: json("courses_granted").$type<APCoursesGrantedTree>().notNull(),
 });
 
+export const libraryTraffic = pgTable(
+  "library_traffic",
+  {
+    id: integer("id").primaryKey(),
+    locationName: varchar("location_name").notNull(),
+    trafficCount: integer("traffic_count").notNull(),
+    trafficPercentage: decimal("traffic_percentage").notNull(),
+    timestamp: timestamp("timestamp").notNull(),
+  },
+  (table) => [index().on(table.id), index().on(table.locationName)],
```

Comment:
you should not explicitly redeclare an index on a primary key (this is always created for you)